scipy/stats/_discrete_distns.py

```diff
@@ -821,17 +821,23 @@ def _stats(self, low, high):
 
     def _rvs(self, low, high):
         """An array of *size* random integers >= ``low`` and < ``high``."""
+        if hasattr(self._random_state, 'integers'):
+            # isinstance(self._random_state, np.random.Generator)
```

Comment:
With the change in numpy random is it worthwhile to add a convenience function over with`check_random_state()` in `_lib._util` ? Not a blocker though.

Reply:
Unfortunately `Generator` and `RandomState` don't have a method with the same name that produces integers. There was only one usage so I didn't write a shim class.

Reply:
it looks like it only shows up in 1 spot in the PR so not a blocker, if we start to see this across PRs we may want to reconsider.